tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -103,22 +103,46 @@ def test_and_boolean_condition():
     assert metric == expected
 
 
-def test_complex_and_condition():
-    query = "geo.country_code:=AT http.method:=GET release:=a transaction.op:=b transaction.status:=aborted transaction.duration:>1s"
+def test_nested_conditions():
```

Comment:
Can you also add a test for precedence like `transaction.duration:>1s OR release:=a AND transaction.op:=b`?

Reply:
Yes definitely, I'd like to write those as unit tests of the converter.